scripts/_autoloads/critters.gd

```diff
@@ -0,0 +1,104 @@
+extends Node
+
+## Critters handles debug UI
+## UI Components
+@export_group("UI Components")
+@export var header_panel: PanelContainer
+@export var footer_panel: PanelContainer
+@export var grid_panels: Array[PanelContainer] = []
+@export var grid_labels: Array[RichTextLabel] = []
+@export var footer_left_label: RichTextLabel
+@export var footer_center_label: RichTextLabel
+@export var footer_right_label: RichTextLabel
+@export_group("Alpha Options")
+@export_range(0.0, 1.0, 0.1) var main_alpha: float = 0.5
+@export_range(0.0, 1.0, 0.1) var header_alpha: float = 1.0
```

Comment:
Static typing can be inferred for floats

Reply:
I made these implicit.